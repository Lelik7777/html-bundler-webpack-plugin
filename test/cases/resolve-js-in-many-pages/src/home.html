<!DOCTYPE html>
<html>
<head>
  <title>Home</title>
  <script src="./common.js" defer="defer"></script>
  <script src="./a.js" defer="defer"></script>
</head>
<body>
  <h1>Home</h1>
</body>
</html>