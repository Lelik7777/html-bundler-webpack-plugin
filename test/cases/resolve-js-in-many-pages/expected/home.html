<!DOCTYPE html>
<html>
<head>
  <title>Home</title>
  <script src="js/common.64204441.js" defer="defer"></script>
  <script src="js/a.0132c52e.js" defer="defer"></script>
</head>
<body>
  <h1>Home</h1>
</body>
</html>